fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/LinkHeaderProvider.java

```diff
@@ -17,33 +17,31 @@
  */
 package org.fcrepo.auth.webac;
 
-import static org.fcrepo.auth.webac.URIConstants.WEBAC_ACCESS_CONTROL_VALUE;
```

Comment:
Is there any way of turning off the unnecessary reordering of `import` statements? It clutters the PR.

Reply:
Sure - I'll try to do that.